docs/premium-features/custom-types-and-actions.md

```diff
@@ -6,25 +6,27 @@ slug: /custom-types-and-actions
 
 # Custom Types & Actions
 
-Custom Types & Actions is an enhanced feature that gives you full control over your Data Transfer Objects (DTOs) and APIs. It allows you to automatically generate high-quality, efficient code for your REST and/or GraphQL APIs.
+Custom Types & Actions is an enhanced feature that gives you full control over your Data Transfer Objects (DTOs) and APIs. It allows you to automatically generate high-quality, efficient code for both the default CRUD operations and your custom APIs that utilize custom DTOs, all within your REST and/or GraphQL APIs.
 
-Custom Types & Actions let you define custom data structures, create API endpoints, and manage the relationships between them, all within the Amplication UI.
+Custom Types & Actions let you define custom data structures, create API endpoints, and manage the relationships between them, all within the Amplication UI. These changes are automatically translated into production-ready code in your repository, ensuring the best quality and ,efficiency.
 
 :::note
-The Custom Types & Actions feature is available for Amplication's **Enterprise** plan.
+The Custom Types & Actions feature is available for Amplication's **Enterprise** plan, including trial plans.
 :::
 
 ## Accessing Custom Types & Actions
 
-To access the Custom Types & Actions feature, navigate to the new **APIs** tab in your Amplication dashboard. This tab provides a centralized view of all your project's Modules, DTOs, and Actions.
+To access the Custom Types & Actions feature, navigate to the new **APIs** tab in your Amplication dashboard.
 
-The APIs tab is divided into sections for each Module, making it easy to browse and manage your application's custom types and actions.
+The APIs tab provides a centralized view of all your project's Modules, DTOs, and Actions. It's also further divided into sections for each Module, making it easy to browse and manage your project's custom types and actions.
 
 ![APIs Tab Overview](./assets/custom-types-and-actions/api-tab-overview.png)
 
 ## Modules
 
-Modules are organizational units that group related DTOs and Actions together. They help structure your application in a logical and maintainable way. Modules are closely tied to entities in your application, allowing you to keep related functionality together.
+Modules are organizational units that group related DTOs and Actions together. They help structure your application in a logical and maintainable way.
+
+Modules are closely tied to entities in your application, with each entity being generated as a Module containing all the entity's code (DTOs, APIs, and more). Each Module is generated as a folder containing all the DTOs and APIs code related to that Module. You can also create additional Modules based on your business needs and requirements.
```

Comment:
It seems to me like we have some kind of redundant here- "each entity being generated as a Module containing all the entity's code (DTOs, APIs, and more)" and "Each Module is generated as a folder containing all the DTOs and APIs code related to that Module". Can we unite it into a single clearer sentence?